style-gutenburg.scss

```diff
@@ -3,3 +3,5 @@
    color: red;
  }
 }
+
+@import 'src/**/*.scss';
```

Comment:
This is already in this file not sure why its not picking it up. On the compare.